src/client/src/orangehrmBuzzPlugin/components/UpcomingAnniversaries.vue

```diff
@@ -19,97 +19,75 @@
  -->
 
 <template>
```

Comment:
@ManulMax I've refactored the anniversary component with less styles and support for tabs. please review.